README.md

```diff
@@ -1,16 +1,39 @@
-razerType is a HacktoberFest React app of a type speed illustrator.
-the purpose of the app revolves around successfully calculating and displaying the typing speed and the accuracy of the user.
 
-The typing test begins as soon as the user clicks on "Start" in the home page. 
+# Hacktoberfest 2022 with IEEE-VIT!
 
-Clicking on "Start" would route the user to the typing area which is composed of 2 text-areas, one overlapped over the other.
+RazorType is a HacktoberFest React app of a type speed illustrator.
 
-The one in the backdrop shall contain a "read-only placeholder" which is supposed to contain the text to be typed.
+The purpose of the app revolves around successfully calculating and displaying the typing speed and the accuracy of the user.
 
-This placeholder would be fetched by a custom function which would generate a paragraph from a .json file filled with words.
+Celebrate open-source by contributing to win goodies! <3
```

Comment:
This can be better phrased. It's also about the spirit of open source and not just goodies :)